src/components/header/Header.js

```diff
@@ -264,7 +252,22 @@ export function Header({
 
                   <button
                     className={`${styles.button}`}
-                    type="submit"
+                    onClick={(e) => {
+                      if (showInfoTooltip || showPopover) {
```

Comment:
det forstår jeg ikke - er det ikke netop det den IKKE skal ??

Reply:
Første gang man trykker skal den ikke gøre noget. showInfoTooltip og showInfoTooltip bliver sat til false når man trykker uden for popover. Så næste gang bliver de false. Og så kan man søge